test/companies/schema/industry_test.exs

```diff
@@ -1,24 +1,22 @@
 defmodule Companies.Schema.IndustryTest do
   @moduledoc """
-    This module holds the unit tests to the industry changeset
+  This module holds the unit tests to the industry changeset
   """
   use Companies.DataCase
 
-  alias Companies.{
-    Schema.Industry
-  }
+  alias Companies.Schema.Industry
 
-  @valid_industry %{
```

Comment:
Some of these attributes used `_params` and others didn't, I made them consistent.